src/callback/class-run-sniffer-callback.php

```diff
@@ -349,73 +349,76 @@ function( $standard ) use ( $standards ) {
 			$selected_standards
 		);
 
-		$theme     = \wp_get_theme( $theme_slug );
-		$php_files = $theme->get_files( 'php', 4, false );
+		$args = [];
 
 		// Current theme text domain.
-		$args[ self::TEXT_DOMAINS ][] = $theme_slug;
+		$args[ self::TEXT_DOMAINS ] = [ $theme_slug ];
 
-		// Frameworks.
-		foreach ( $php_files as $key => $file ) {
-			if ( strrpos( $key, 'hybrid.php' ) ) {
-				$args[ self::TEXT_DOMAINS ][] = 'hybrid-core';
-			}
-			if ( strrpos( $key, 'kirki.php' ) ) {
-				$args[ self::TEXT_DOMAINS ][] = 'kirki';
-			}
-		}
+		$all_files = [ 'php' ];
 
-		if ( $check_php_only ) {
-			$all_files = $theme->get_files( [ 'php' ], -1, false );
-		} else {
-			$all_files = $theme->get_files( [ 'php', 'css,', 'js' ], -1, false );
+		if ( ! $check_php_only ) {
+			$all_files = array_merge( $all_files, [ 'css', 'js' ] );
 		}
 
-		$removed_files = [];
+		$theme     = wp_get_theme( $theme_slug );
+		$all_files = $theme->get_files( $all_files, -1, false );
 
 		/**
 		 * Check if a file is minified.
 		 * Loops through all the files and checks if it's minified. If it is, skip it
 		 * because it will break phpcs.
 		 */
 		foreach ( $all_files as $file_name => $file_path ) {
-			// Check if files have .min in the file name.
-			if ( strpos( $file_name, '.min.js' ) !== false || strpos( $file_name, '.min.css' ) !== false ) {
-				unset( $all_files[ $file_name ] );
-				$removed_files[] = $file_name;
-				break;
+
+			// Check for Frameworks.
+			$allowed_frameworks = [
```

Comment:
Like how this is extracted, makes it easier to add later on. Plus we can add a filter after the array like

```php
$allowed_frameworks = apply_filters( 'theme_sniffer_add_allowed_frameworks', $allowed_frameworks );
```

That way we can make the plugin more extendable.

Thoughts?

Reply:
yeah - I thought about a filter there, but I also don't know if it should be there or not.  I wouldn't want themes to push something in there just to bypass requirements if a reviewer is relying on the plugin over cli.  I mean technically I guess they could get tricky and just use some css or js to hide errors if they wanted, but I dunno - that's up to you :P

Reply:
Well I don't think it's a bad addition, because if one day the theme sniffer will be used instead of theme check, they won't be able to bypass it 🙂 
What authors do on their own sites/locally shouldn't concern us imo